### PR TITLE
Update private rules

### PR DESCRIPTION
**Pull request by adamkvitek**
Resolves https://linear.app/r2c/issue/MKT-378#comment-32996818

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/writing-rules/private-rules.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Private rules
 
-Users in the [Team and Enterprise tier](https://semgrep.dev/pricing) for Semgrep App can publish rules to the Semgrep Registry as private rules that are not visible to others outside their organization. The private rules enable you to hide code-sensitive information or legal requirements that prevent you from using a public registry.
+Private rules enable you to publish rules so only users in your organization can see them but these rules are not displayed externally. Users in the [Team and Enterprise tier](https://semgrep.dev/pricing) for Semgrep App can publish rules to the Semgrep Registry as private rules that are not visible to others outside their organization. The private rules enable you to hide code-sensitive information or legal requirements that prevent you from using a public registry.
```
Comment (efortuna) [inline docs/writing-rules/private-rules.md:10]: I'd remove this change, as you don't necessarily have to "publish" private rules

File: docs/writing-rules/private-rules.md [hunk 2]
```diff
@@ -45,13 +45,12 @@ You can also find it in the [registry](https://semgrep.dev/r) by searching for [
 
 To enforce the rule on new scans, add the rule in the [registry](https://semgrep.dev/r) to an existing policy.
 
-## Automatically publishing rules
+## Automatically publishing private rules
 
 This section provides a sample of GitHub Actions workflow to automatically publish rules from a private Git repository after a merge to the `main`, `master`, or `develop` branches.
```
Comment (efortuna) [inline docs/writing-rules/private-rules.md:50]:
```suggestion
This section provides a sample of the GitHub Actions workflow to automatically publish your private rules so they are accessible within your private organization. "Publishing" your private rules in this manner does not make them public. This example publishes from a private Git repository after a merge to the `main`, `master`, or `develop` branches.
```

File: docs/writing-rules/private-rules.md [hunk 3]
```diff
@@ -45,13 +45,12 @@ You can also find it in the [registry](https://semgrep.dev/r) by searching for [
 
 To enforce the rule on new scans, add the rule in the [registry](https://semgrep.dev/r) to an existing policy.
 
-## Automatically publishing rules
+## Automatically publishing private rules
```
Comment (efortuna) [inline docs/writing-rules/private-rules.md:48]: this change is probably unnecessary

Comment (adamkvitek) [reply docs/writing-rules/private-rules.md:48]: Ok!

File: docs/writing-rules/private-rules.md (hunk 1 above)
Comment (adamkvitek) [reply docs/writing-rules/private-rules.md:10]: Removing))

File: docs/writing-rules/private-rules.md (hunk 2 above)
Comment (adamkvitek) [reply docs/writing-rules/private-rules.md:50]: Ok, taking inspiration, changing to: 

```markdown
This section provides an example of how to automatically publish your private rules so they are accessible within your private organization. "Publishing" your private rules in this manner does not make them public. The following sample of the GitHub Actions workflow publishes rules from a private Git repository after a merge to the `main`, `master`, or `develop` branches.
```

**Review by efortuna — APPROVED** [review]
LGTM!

**Review by ievans — APPROVED** [review]

**Pull request merged**